docs/src/analysis/sensitivity.md

```diff
@@ -604,21 +604,22 @@ In this case we'd like to calculate the adjoint sensitivity of the scalar energy
 functional
 
 ```math
-G(u,p)=\int_{0}^{T}\frac{\sum_{i=1}^{n}u_{i}^{2}(t)}{2}dt
+G(u,p)=\int_{0}^{T}\frac{\sum_{i=1}^{3}u_{i}^{2}(t)}{2}dt
 ```
 
 which is
 
 ```julia
-g(u,p,t) = (sum(u).^2) ./ 2
+g(u,p,t) = (sum(u).^2) / 2
```

Comment:
`sum(u).^2` -> `sum(u)^2`

Reply:
To rebuild the function under the integral in G you even need  
sum(u.^2)/2 
Sorry there, my bad.